wfe2/wfe.go

```diff
@@ -1419,6 +1457,16 @@ func (wfe *WebFrontEndImpl) updateAccount(
 		return nil, probs.ServerInternal("Error updating account")
 	}
 
+	emails := contactsToEmails(accountUpdateRequest.Contact)
+	if wfe.ee != nil && len(emails) > 0 {
+		_, err := wfe.ee.SendContacts(ctx, &emailpb.SendContactsRequest{
+			Emails: emails,
+		})
+		if err != nil {
+			wfe.log.Warningf("Error creating contact: %v", err)
+		}
```

Comment:
Same comment as above: we should mainly be returning errors so we don't mask internal failures.